### ISSUE TITLE
Colors for test explorer looks strange on default light theme

### ISSUE THREAD

**Issue by andreamah** [microsoft/vscode#163678]
<img width="488" alt="image" src="https://user-images.githubusercontent.com/31675041/195913129-f4b2f92e-ce34-490f-8314-25c179673301.png">

The selected test has a dark blue background with black text, which makes it hard to read.
